README.md

```diff
@@ -94,8 +94,8 @@ client will automatically find and trust it.
 
 ## Development
 
-The quinn-proto test suite uses simulated IO for reproducibility and to avoid
-long sleeps in certain timing-sensitive tests. If the `SSLKEYLOGFILE`
+The Quinn-proto test suite uses simulated IO for reproducibility and to avoid
```

Comment:
The crate is spelled with a lower-case q.

Reply:
Of course, I saw Quin also, however, a crate is not capitalized but the name is. Fil fix.

Reply:
Yeah, the project we've usually spelled with a capital. But for crates it doesn't make much sense.